contracts/cw-governance-interface/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod voting;
```

Comment:
This module should be, rather a package like https://github.com/CosmWasm/cw-plus/tree/main/packages/cw20/src